apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/entity/condition/CodeToCondition.java

```diff
@@ -0,0 +1,60 @@
+package gov.cdc.nbs.questionbank.entity.condition;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.persistence.Entity;
+import java.time.Instant;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "Code_to_condition", catalog="NBS_SRTE")
+public class CodeToCondition {
+    @EmbeddedId
+    private CodeToConditionId id;
```

Comment:
I don't see this object being defined anywhere. Also, how does the `Code_to_condition` table work? What is it used for? 

When I create a condition through the legacy UI, no entry is made into the `Code_to_condition` table.